bin/parse_pom_xml

```diff
@@ -22,17 +22,28 @@ if parsed_file.errors.size.positive?
   exit(1)
 end
 
-dependencies = []
+properties = {}
+# Parse contents within each <properties> tag
+nodes = parsed_file.css('properties')
```

Comment:
If `properties` does not exist, then will `nodes` be empty array or `nil`?
If `nil`, then the `node.each` will throw an exception.

Reply:
If `<properties>` is absent then returns an empty array so should be good.